.fixtures.yaml

```diff
@@ -0,0 +1,10 @@
+fixtures:
+  forge_modules:
+    apt: 'puppetlabs/apt'
+    epel: 'puppet/epel'
+    logrotate: 'puppet/logrotate'
+    stdlib: 'puppetlabs/stdlib'
+  repositories:
+    yumrepo_core:
+      repo: https://github.com/puppetlabs/puppetlabs-yumrepo_core.git
+      puppet_version: ">= 6.0.0"
```

Comment:
at vox we do not add those constraints anymore because we do not test on Puppet 5 anymore. you can treat yumrepo_core like the other dependencies